src/App.tsx

```diff
@@ -15,36 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabet,
+  length,
+}
+
+interface SortParams {
+  sortBy: number | null;
+  isReverse: boolean;
+}
+
+const getPreparedGoods = (goods: string[], sorter: SortParams) => {
+  const { sortBy, isReverse } = sorter;
+  const prepGoods = [...goods];
+
+  if (sortBy === SortType.alphabet) {
+    prepGoods.sort((a, b) => a.localeCompare(b));
+  }
+
+  if (sortBy === SortType.length) {
+    prepGoods.sort((a, b) => a.length - b.length);
+  }
+
+  if (isReverse) {
+    prepGoods.reverse();
+  }
+
+  return prepGoods;
+};
+
 export const App: React.FC = () => {
+  const [sortBy, setSortBy] = useState<number | null>(null);
```

Comment:
The initial state of `sortBy` is set to `null`. Consider initializing it with one of the `SortType` enum values (e.g., `SortType.alphabet`) to maintain consistency with the sorting logic.